pom.xml

```diff
@@ -801,37 +801,37 @@
             <dependency>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>org.apache.felix.framework</artifactId>
-                <version>4.4.1</version>
+                <version>7.0.1</version>
```

Comment:
This likely won't work  due to changes in the Felix project between 4.x and 7.x.

Regarding the dependencies updates (Guice, etc.), is there any of them that are *must* have? If so, let's limit the changes to these only.